src/http/api/users.js

```diff
@@ -0,0 +1,53 @@
+const { Route, EndpointUtils } = require('../../index')
+const { Router } = require('express')
+
+const DAILY_INTERVAL = 24 * 60 * 60 * 1000 // 1 day
+const DAILY_BONUS = () => Math.ceil(Math.random() * 2000) + 750
+
+module.exports = class Users extends Route {
+  constructor (client) {
+    super(client)
+    this.name = 'users'
+    this.subRoutes = [new Economy(client, this)]
+  }
+}
+
+class Economy extends Route {
+  constructor (client, parentRoute) {
+    super(client, parentRoute)
+    this.name = 'economy'
+  }
+
+  register (app) {
+    const router = Router()
+
+    // Daily
+    router.post('/daily/:id', EndpointUtils.authenticate(), EndpointUtils.handleUser(this), async (req, res) => {
+      const id = req.id
+
+      const doc = await this.client.database.users.get(id)
```

Comment:
Don't use the database directly, use modules!!!1!

Reply:
Also, why are dailies here? I don't think they should be claimable through the API.

Reply:
old diff pls update kthx